### DISCUSSION TITLE
Front-end bot do Twitter

### DISCUSSION THREAD

**Discussion by germanolira** [frontendbr/forum#2078] [Dicas]
Olá, teria como adicionar a tag da vaga, júnior, pleno ou sênior, quando o bot for postar no Twitter?

[
![image](https://user-images.githubusercontent.com/25235170/117705070-f698eb80-b1a1-11eb-8b7c-a7ee462aecb3.png)
](url)

Exemplo: [Remoto] Front-end Developer na ProJuris [Sênior]

Fica muito melhor, porque seria um filtro automático

**Comment by wilsonneto-swe** [comment]
Qual bot? Feito em qual linguagem? Vc que fez? Qual lib? Qual a sua dúvida? rs

Você está perguntando se tem como colocar uma tag a mais no seu bot? Você está com dúvida referente  libe do bot? Isto é uma sugestão?

**Reply by germanolira** [reply]
Esse bot https://twitter.com/frontendbrvagas, seria interessante adicionar mais tags no título da postagem do Twitter
